File: src/main/java/org/example/gogoma/domain/user/dto/CreateUserRequest.java
package org.example.gogoma.domain.user.dto;

import lombok.Data;
import org.example.gogoma.domain.user.enums.Gender;

@Data
public class CreateUserRequest {

    private Long kakaoId;
    private String name;
    private String profileImage;
    private String email;
    private Gender gender;
    private String birthDate;
    private String birthYear;
    private String phoneNumber;
    private String roadAddress;
    private String detailAddress;
    private String clothingSize;
    private String fcmToken;
}

File: app/src/main/java/com/example/gogoma/data/model/CreateUserRequest.kt
package com.example.gogoma.data.model

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class CreateUserRequest (
    val kakaoId: Long,
    val name: String,
    val profileImage: String,
    val email: String,
    val gender: String, // 원래 Enum이지만, 여기서는 string 처리
    val birthDate: String,
    val birthYear: String,
    val phoneNumber: String,
    val roadAddress: String,
    val detailAddress: String,
    val clothingSize: String
) : Parcelable